CHANGELOG.rst

```diff
@@ -1,4 +1,4 @@
-1.8.0 (UNRELEASED)
+1.8.0 (2020-01-27)
```

Comment:
2021 :wink:

Took the freedom to fix it directly in ed8b28a7198726165e62a36e18c903013b3a6b04.

Reply:
Hah! 😆 

Thanks!